test/unit/YieldTokenIncidents/redeemPayout.js

```diff
@@ -588,11 +705,12 @@ describe('redeemPayout', function () {
       [segment],
     );
 
+    const priceBefore = parseEther('1.1');
     {
       const { timestamp: currentTime } = await ethers.provider.getBlock('latest');
       await yieldTokenIncidents
         .connect(advisoryBoard)
-        .submitIncident(4, parseEther('1.1'), currentTime + segment.period / 2, parseEther('100'), '');
+        .submitIncident(4, priceBefore, currentTime + segment.period / 2, parseEther('100'), '');
```

Comment:
make a named constant productId = 4

that should be used in the mockCover as well